### PR TITLE
Fix plot behavior when using keywords screenshot and off_screen

### PR DESCRIPTION
**Pull request by PProfizi**
Issue #193 
Fixed the behavior to have what would be expected.

Still have to add tests, update the docstrings (some arguments are now kwargs), and update the examples.

PS: I have modified the Plotter class, while work is underway to deprecate this and use DpfPlotter instead. This behavior should be copied when migrating plot_contour to the DpfPlotter.

PPS: when initializing a Plotter directly and then calling plot_contour from it, the off_screen keyword must be given when creating the Plotter as it will be an attribute of this Plotter instance, whereas the screenshot keyword should be given to plot_contour (which is logical but should be explained).

### CODE REVIEW

**Comment by akaszynski** [comment]
Rerunning ci by closing/opening

File: ansys/dpf/core/meshed_region.py [hunk 1]
```diff
@@ -406,13 +406,18 @@ def plot(
...
+        pl = _DpfPlotter(self, notebook=notebook, **kwargs)
+        kwargs["screenshot"] = screenshot
+        kwargs["text"] = text
         if field_or_fields_container is not None:
             return pl.plot_contour(
                 field_or_fields_container,
                 notebook,
                 shell_layers,
-                off_screen,
+                # off_screen,
```
Comment (anslpa) [inline ansys/dpf/core/meshed_region.py:420]: @PProfizi loose of compatibility with potential user scripts ? Is it wanted ?

**Comment by PProfizi** [comment]
All plotting has been moved to the new DpfPlotter for 3D plotting.
Still have to modify/improve the plot_chart functionality.

File: ansys/dpf/core/meshed_region.py [hunk 2]
```diff
@@ -406,20 +406,18 @@ def plot(
...
-                show_axes,
-                **kwargs
-            )
+            pl = Plotter(self, **kwargs)
+            return pl.plot_contour(field_or_fields_container, notebook,
+                                   shell_layers, off_screen, show_axes, **kwargs)
 
-        # otherwise, simply plot self
+        # otherwise, simply plot the mesh
+        kwargs["off_screen"] = off_screen
```
Comment (cbellot000) [inline ansys/dpf/core/meshed_region.py:415]: can we not remove that?

**Review by cbellot000 — APPROVED** [review]

**Pull request merged**